[fix] prisma @index 수정

## 🤔 오류 내용

> [app 7/7] RUN npx prisma generate:
0.489 Prisma schema loaded from prisma/schema.prisma
0.496 Error: Prisma schema validation - (get-dmmf wasm)
0.496 Error code: P1012
0.496 error: Attribute not known: "@index".
0.496   -->  prisma/schema.prisma:36
0.496    | 
0.496 35 |   id           Int      @id @default(autoincrement())
0.496 36 |   clerk_id     String   @index
0.496    | 
0.496 error: Attribute not known: "@index".
0.496   -->  prisma/schema.prisma:68
0.496    | 
0.496 67 |   id          Int           @id @default(autoincrement())
0.496 68 |   chatroom_id Int   @index
0.496    | 
0.496 
0.496 Validation Error Count: 2
0.496 [Context: getDmmf]
0.496 
0.496 Prisma CLI Version : 6.12.0
------
failed to solve: process "/bin/sh -c npx prisma generate" did not complete successfully: exit code: 1
[ERROR] 빌드 실패! 오류를 확인하세요
<br>

## ⚠ 에러 캡쳐

<br>

--- a/Dockerfile
+++ b/Dockerfile
@@ -15,6 +15,8 @@ RUN if [ "$NODE_ENV" = "production" ]; then npm install --omit=dev; else npm ins
 COPY src ./src
 COPY prisma ./prisma
 
+RUN npx prisma generate
+
 # (테스트, 개발용 파일/폴더는 .dockerignore로 제외)
 
 # 앱 실행 (필요에 따라 수정)

--- a/prisma/schema.prisma
+++ b/prisma/schema.prisma
@@ -33,7 +33,7 @@ model User {
 
 model Persona {
   id           Int      @id @default(autoincrement())
-  clerk_id     String   @index
+  clerk_id     String
   user         User     @relation(fields: [clerk_id], references: [clerk_id], onDelete: Cascade)
   name         String   @db.VarChar(100)
   image_url    String   @db.VarChar(1024)
@@ -46,6 +46,8 @@ model Persona {
   updated_at   DateTime?
   is_deleted   Boolean  @default(false)
   chatRooms    ChatRoom[]
+
+  @@index([clerk_id])
 }
 
 model ChatRoom {
@@ -65,11 +67,13 @@ model ChatRoom {
 
 model ChatLog {
   id          Int           @id @default(autoincrement())
-  chatroom_id Int   @index
+  chatroom_id Int
   chatRoom    ChatRoom @relation(fields: [chatroom_id], references: [id], onDelete: Cascade)
   text        String
   type        ChatLogType
   speaker     ChatLogSpeaker
   time        DateTime @default(now())
   is_deleted  Boolean  @default(false)
+
+  @@index([chatroom_id])
 }

